frontend/ui/data/css_tweaks.lua

```diff
@@ -259,6 +259,122 @@ width: 100% !important;
     },
     {
         title = _("Miscellaneous"),
+        {
+            title = _("Alternative TOC hints"),
+            {
+                title = _("About alternative TOC"),
+                smaller_font = true,
+                info_text = _([[
+An alternative table of content can be built with a long-press on the "Table of content" menu item.
+
+It will be built from document headings <H1> to <H6>. Some of these can be ignored with the tweaks available here.
+If the document contains no heading, or all are ignored, the alternative TOC will be built from the document fragments, to point to the start of each individual HTML file in the EPUB.
+
+Hints can be set to other non-heading elements, for them to be used as TOC items, in some user style tweaks (as they would be quite book-specific, see last tweak here for some examples).
```

Comment:
```suggestion
Hints can be set to other non-heading elements, so they can used as TOC items in some user style tweaks. Since this would be quite book-specific, please see the previous tweaks for some examples.
```

Reply:
I edited it to final.